docs/hosting/deployment/index.rst

```diff
@@ -375,3 +381,49 @@ First, start ``parsec`` with the custom CA:
   parsec
 
 After that go to ``Menu``/``Join an organization`` (or ``CTRL+O``) and paste the link from before (should already be filled in the text field). Follow the instructions to create the first user of the organization.
+
+.. _behind_a_proxy:
+
+Running behind a reverse proxy
+******************************
+
+To run Parsec behind a reverse proxy you will need to add the option ``--proxy-trusted-address`` or set the environment variable ``PARSEC_PROXY_TRUSTED_ADDRESS`` to the address of the reverse proxy (e.g.: ``localhost``).
+
+.. tip::
+
+  You can provide multiple addresses by separating them with a comma.
+
+  Example: Use the option ``--proxy-trusted-address '::1,10.0.0.42'`` will trust the address ``::1`` and ``10.0.0.42``
+
+An example of a reverse proxy configuration for ``nginx`` can be found in :ref:`the docker compose file <the-docker-compose-file>`:
```

Comment:
I have that rendering when using github view file option 
![image](https://github.com/user-attachments/assets/63c6a1d7-bf08-4c45-b961-cd6331a36194)

Reply:
:sweat_smile: So GitHub rendering of RST document is not perfect.

It's actually rendered like so:

![image](https://github.com/user-attachments/assets/231f3511-b03f-4f24-a00d-79e2082a1c26)

FYI, we have RTD that run on each PR to generate the documentation specific for that PR.
For this one, you can check it out at <https://parsec-cloud--8676.org.readthedocs.build/en/8676/hosting/deployment/index.html>

To get the link, you click on the `Details` element of `docs/readthedocs.org:parsec-cloud` of the PR status list:

![image](https://github.com/user-attachments/assets/91b2393a-c815-4076-b754-3e96c6d29ad8)